src/renderer/models/account.ts

```diff
@@ -60,8 +61,32 @@ export class CGAccount implements IAccount {
     /**
      * Returns array of beacon nodes that validator uses.
      */
-    public getValidatorBeaconNodes(validatorAddress: string): BeaconNode[] {
-        return this.validatorsBeaconNodes[validatorAddress];
+    public async getValidatorBeaconNodes(validatorAddress: string): Promise<BeaconNode[]> {
```

Comment:
I think this should return IEth2ApiClient and let container generate required data from that

Reply:
Can't do like that because of other beacon node info I need but will pull out something.

Reply:
Add required stuff to IEth2ApiClient?

Reply:
or include IEth2ApiClient in BeaconNode type